### PR TITLE
Update Testimonials and Images

### PR DESCRIPTION
**Pull request by ksatx**

### CODE REVIEW

**Comment by ksatx** [comment]
[Swap Gil with Tim quote Tim Slagle, Developer Advocacy Manager, Zoom "On average, Stoplight saves our team 15-20 hrs per week." Have Link that says, "Watch video"](https://trello.com/c/3qnI43vh/243-swap-gil-with-tim-quote-tim-slagle-developer-advocacy-manager-zoom-on-average-stoplight-saves-our-team-15-20-hrs-per-week-have-l)

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 5b7849235a4e383ac9129ce9d296b0e6423e8ff1

https://deploy-preview-1257--stoplightio.netlify.com

File: netlify/pages/home.yaml [hunk 1]
```diff
@@ -42,13 +42,12 @@ testimonials:
         development! We love all the rich design, documentation and testing
         features, which has enabled us to design awesome APIs for our customers.
       role: Product Manager
-    - author: Gil Barbara
-      company: AMARO
-      image: /images/testimonials/gil.jpeg
+    - author: Tim Slagle
+      company: Zoom
+      image: \images\tim s.jpg
```
Comment (rowa97) [inline netlify/pages/home.yaml]: this image filename is improperly formatted, space between tim and s

File: netlify/forms/contact-us.yaml [hunk 2]
```diff
@@ -43,13 +43,12 @@ their API & Microservices workflow'
       alt: Zendesk
 testimonials:
   testimonials:
-    - author: Kin Lane
-      image: /images/testimonials/kin_lane.jpeg
+    - author: Tim Slagle
+      company: Zoom
+      image: \images\tim s.jpg
```
Comment (rowa97) [inline netlify/forms/contact-us.yaml]: image filename has a spacae

**Pull request merged**